ui-tests/tests/nbdime-merge-test10.spec.ts

```diff
@@ -0,0 +1,28 @@
+import { expect, test } from '@playwright/test';
+
+test.beforeEach(async ({ page }) => {
+  await page.goto('http://localhost:41000/merge');
+  await page.locator('#merge-local').fill('data/merge_test10/left.ipynb');
+  await page.locator('#merge-base').fill('data/merge_test10/center.ipynb');
+  await page.locator('#merge-remote').fill('data/merge_test10/right.ipynb');
+  await page.getByRole('button', { name: 'Merge files' }).click();
+});
+
+/* notebooks of same length and 1 conflict*/
+test.describe('merge test10', () => {
```

Comment:
```suggestion
test.describe('merge test4', () => {
```

Also the indexation of examples could start from 1.